sugoi-api-ldap-store-provider/src/main/resources/application.properties

```diff
@@ -9,4 +9,5 @@ fr.insee.sugoi.ldap.default.password=
 fr.insee.sugoi.ldap.default.pool=
 fr.insee.sugoi.ldap.default.port=
 fr.insee.sugoi.ldap.default.group_source_pattern=
-fr.insee.sugoi.ldap.default.group_filter_pattern=
\ No newline at end of file
+fr.insee.sugoi.ldap.default.group_filter_pattern=
+fr.insee.sugoi.ldap.default.app_managed_attribute_pattern=
```

Comment:
Can you add it as well in sugoi-api-distribution/sugoi-api-distribution-full-env/src/main/resources/tomcat-properties/tomcat1.properties for testing purposes ?